util/src/test/java/io/camunda/zeebe/util/jar/ExternalJarClassLoaderTest.java

```diff
@@ -52,4 +56,26 @@ void shouldUseSystemClassLoaderAsFallback(final @TempDir File tempDir)
         .isEqualTo(getClass().getClassLoader())
         .isEqualTo(ClassLoader.getSystemClassLoader());
   }
+
+  @Test
+  void shouldLoadResourceFiles(final @TempDir File tempDir) throws Exception {
+    final String testResourceValue = "test-value";
+    final String resourceName = "test-resource.txt";
+    final File jarFile = new File(tempDir, "with-resource.jar");
+    try (final FileOutputStream fileOutputStream = new FileOutputStream(jarFile)) {
+      try (final JarOutputStream jarOutputStream = new JarOutputStream(fileOutputStream)) {
+        jarOutputStream.putNextEntry(new JarEntry(resourceName));
+        jarOutputStream.write(testResourceValue.getBytes(StandardCharsets.UTF_8));
+        jarOutputStream.closeEntry();
+      }
+    }
+    final var classLoader = ExternalJarClassLoader.ofPath(jarFile.toPath());
```

Comment:
:wrench: I would use var consistently. I think rule of thumb is use var when the type is obvious. So most of the declarations above, e.g. `final FileOutputStream fileOutputStream = new FileOutputStream`. In this test, all of them are obvious.

We don't have a clear rule though, so the only thing I would suggest is to stay consistent within a single code unit (either use var or don't :slightly_smiling_face:)

Reply:
@npepinpe Hey, it's done :)